featurebyte/service/feature_materialize.py

```diff
@@ -537,7 +538,8 @@ async def _get_feast_feature_store(self) -> Optional[FeastFeatureStore]:
         feast_registry = await self.feast_registry_service.get_feast_registry_for_catalog()
         if feast_registry is None:
             return None
-        return await self.feast_feature_store_service.get_feast_feature_store(feast_registry.id)
+        assert isinstance(feast_registry, FeastRegistryModel)
+        return await self.feast_feature_store_service.get_feast_feature_store(feast_registry)
```

Comment:
By making this changes, get online features' runtime is reduced by 1s for a feature list with 306 features (14s -> 13s).